llama-index-integrations/llms/llama-index-llms-monsterapi/README.md

```diff
@@ -1 +1,53 @@
 # LlamaIndex Llms Integration: Monsterapi
+
+MonsterAPI LLM.
+
+Monster Deploy enables you to host any vLLM supported large language model (LLM) like Tinyllama, Mixtral, Phi-2 etc as a rest API endpoint on MonsterAPI's cost optimised GPU cloud.
+
+With MonsterAPI's integration in Llama index, you can use your deployed LLM API endpoints to create RAG system or RAG bot for use cases such as: - Answering questions on your documents - Improving the content of your documents - Finding context of importance in your documents
+
+Once deployment is launched use the base_url and api_auth_token once deployment is live and use them below.
+
+Note: When using LLama index to access Monster Deploy LLMs, you need to create a prompt with required template and send compiled prompt as input.
+
+See `LLama Index Prompt Template Usage example` section for more details.
+
+see (https://developer.monsterapi.ai/docs/monster-deploy-beta) for more details
+
+Once deployment is launched use the base_url and api_auth_token once deployment is live and use them below.
+
+Note: When using LLama index to access Monster Deploy LLMs, you need to create a prompt with reqhired template and send compiled prompt as input. see section `LLama Index Prompt Template
+    Usage example` for more details.
+
+Examples:
+
+`pip install llama-index-llms-monsterapi`
+
+1.  MonsterAPI Private LLM Deployment use case
+
+    ````python
+    from llama_index.llms.monsterapi import MonsterLLM
+
+        llm = MonsterLLM(
```

Comment:
Small correction needed here: 
base_url -> api_base
monster_api_key -> api_key

Reply:
Good catch! This is basically the only thing I noticed as well

Reply:
Done, updated! Good catch!

Reply:
@logan-markewich @anoopshrma Can you guys look at the CI test failing and help me with it? we don't have monsterapi package dependency anymore. I updated the pyproject.toml accordingly not sure where the reference of monsterapi is coming in!

Reply:
@logan-markewich @anoopshrma Need a little help here!